app/services/content/item_details_service.rb

```diff
@@ -0,0 +1,13 @@
+require 'gds_api/content_store'
+
+class Content::ItemDetailsService
```

Comment:
`ItemDetails` doesn't feel like what this object does. 
I would add another method to the `ItemsService` to retrieve the `raw_json` from the `ContentStore`